packages/cosmic-swingset/tools/test-bridge-utils.ts

```diff
@@ -0,0 +1,77 @@
+import { BridgeId, VBankAccount } from '@agoric/internal';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import { makeTestAddress } from '@agoric/orchestration/tools/make-test-address.js';
+import { fakeLocalChainBridgeTxMsgHandler } from '@agoric/vats/tools/fake-bridge.js';
+import { Fail, q } from '@endo/errors';
+
+type MockBridgeOptions = {
+  ibcKit: { handleOutboundMessage: (msg: any) => unknown };
+  outboundMessages: Map<BridgeId, any[]>;
+  storageKit: import('@agoric/internal/src/storage-test-utils.js').FakeStorageKit;
+};
+
+export const makeMockBridgeKit = ({
+  ibcKit,
+  outboundMessages = new Map(),
+  storageKit = makeFakeStorageKit(''),
+}: Partial<MockBridgeOptions> = {}) => {
+  let lastBankNonce = 0;
+  let lcaAccountsCreated = 0;
+  let lcaSequenceNonce = 0;
+
+  const handleBridgeSend = (bridgeId: BridgeId, obj: any) => {
```

Comment:
Suggestion: Rename these parameters and narrow the second one (which may require casting in the delegating calls, but that's fine).
```suggestion
  const handleBridgeSend = (
    destPort: BridgeId,
    msg: Record<string, any> & {type: string},
  ) => {
```